exo-deploy/terraform/aws/aws.tf

```diff
@@ -0,0 +1,60 @@
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_availability_zones" "available" {}
+
+module "internal_dns" {
+  source = "./internal-dns"
+
+  name    = "${var.name}.local"
```

Comment:
Seems that the convention is to pass `"${var.env}-${var.name}"` as the name, so this should be `"${var.env}.${var.name}.local"`